### PR TITLE
Params check

### PR DESCRIPTION
**Pull request by purefan**
Just adding some sugar

### CODE REVIEW

File: hangover.sh [hunk 1]
```diff
@@ -1,5 +1,26 @@
 #!/bin/sh -e
 
+# Fill in the blanks
+# Phone numbers with leading country code: +46 012 345 6788
+MY_NUMBER=''
+NUMBER_OF_BOSS=''
+
+# Twilio account information
+TWILIO_ACCOUNT_SID=""
```
Comment (niroyb) [inline hangover.sh]: This will break using environment variables and make it anoying to commit other changes if the values are set.

Comment (richarson) [inline hangover.sh]: How about:

MY_NUMBER=${MY_NUMBER:-}

That way you can set them in the script or in your env?

Comment (richarson) [inline hangover.sh]: Nevermind, I guess leaving those variables commented out with a warning to either set them in your env or modifiyng the script would be saner.

Comment (jirutka) [inline hangover.sh]: > MY_NUMBER=${MY_NUMBER:-XXX}

Shorter and DRYer:

```sh
: ${MY_NUMBER:=MY_NUMBER}
```

**Pull request closed without merging**